packages/web-client/app/routes/index.ts

```diff
@@ -0,0 +1,110 @@
+import Route from '@ember/routing/route';
+
+import CardstackLogo from '../images/icons/cardstack-logo-white-text.svg';
```

Comment:
Is this an example of the surprising pathing that you need to use? Would full paths work, and would that be better? i.e. `@cardstack/web-client/images/icons/cardstack-logo-white-text.svg`

Reply:
That seems better! Thanks @lukemelia.

Reply:
Will also apply to Boxel later